test_conformance/images/kernel_read_write/test_iterations.cpp

```diff
@@ -1008,9 +1008,12 @@ int validate_image_2D_sRGB_results(void *imageValues, void *resultValues, double
                             maxPixel = sample_image_pixel_float_offset( imageValues, imageInfo,
                                                                         xOffsetValues[ j ], yOffsetValues[ j ], 0.0f, norm_offset_x, norm_offset_y, 0.0f,
                                                                         imageSampler, expected, 0, &containsDenormals );
-                        float err1 = fabsf( sRGBmap( resultPtr[0] ) - sRGBmap( expected[0] ) );
-                        float err2 = fabsf( sRGBmap( resultPtr[1] ) - sRGBmap( expected[1] ) );
-                        float err3 = fabsf( sRGBmap( resultPtr[2] ) - sRGBmap( expected[2] ) );
+                        float err1 =
+                            fabs(sRGBmap(resultPtr[0]) - sRGBmap(expected[0]));
+                        float err2 =
+                            fabs(sRGBmap(resultPtr[1]) - sRGBmap(expected[1]));
+                        float err3 =
+                            fabs(sRGBmap(resultPtr[2]) - sRGBmap(expected[2]));
```

Comment:
Hmm, given how many times you've had to make the same change, I'd guess there's a common function to be extracted there :).